waspc/data/Cli/templates/basic/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "prototype",
+  "dependencies": {
+    "wasp": "file:.wasp/out/sdk/wasp",
+    "react": "^18.2.0"
+  },
+  "devDependencies": {
+    "typescript": "^5.1.0",
+    "vite": "^4.3.9",
+    "@types/react": "^18.0.37",
```

Comment:
We're currently getting `types/node` (and possibly other similar packages) for by accident (they're listed as a peer dependency of other dependencies).

Todo: figure out whether we want to include them or not.